frontend/src/containers/Ledger.js

```diff
@@ -214,6 +214,17 @@ export class Ledger extends Component {
   }
 }
 
+function findExpense(expenseId, expensesObj) {
+  if (expensesObj) {
+    for (let key in expensesObj) { // eslint-disable-line prefer-const
```

Comment:
Can't use a const here? 🤔

It's kind of hacky to do it this way. Ideally we would want to keep those expenses in the store with the key being their id. (I'm not asking to change it, just feedback)